src/hotspot/share/jeandle/jeandleCompiledCode.cpp

```diff
@@ -205,6 +205,27 @@ void JeandleCompiledCode::install_obj(std::unique_ptr<ObjectBuffer> obj) {
   }
 }
 
+uint64_t JeandleCompiledCode::estimate_stubs_size() {
+  // starting from 3 to keep a small reserve, 
+  // because the accurate number of static call stubs can only be obtained during the resolve_reloc_info phase.
+  uint64_t num_static_call  = 3;
+  for (auto *site : _non_routine_call_sites) {
+    if (site->type() == JeandleCompiledCall::STATIC_CALL)
+      ++num_static_call ;
+  }
+  // Static-call stubs: AArch64 13 * 4 B, x86 28 B → worst-case 52 B/site 
```

Comment:
Perhaps we can implement `estimate_xxx_size` at the target-specific file like `jeandleCompiled_x86.cpp`, to get more precise sizes.